src/python/string/punctuation.js

```diff
@@ -0,0 +1,8 @@
+// Original by: Clari (https://github.com/Clari-7744)
```

Comment:
Please move the comment to line 3, so it's between the function definition and the `example 1`.

Reply:
done